Source/LibationWinForms/grid/GridEntry.cs

```diff
@@ -160,10 +160,12 @@ private void UserDefinedItem_ItemChanged(object sender, string itemName)
 					break;
 				case nameof(udi.BookStatus):
 					Book.UserDefinedItem.BookStatus = udi.BookStatus;
+					_bookStatus = udi.BookStatus;
```

Comment:
Huh... I could have sworn this was already in there. I hope I didn't take it out by mistake

Reply:
Who knows. This section has been touched so much over the last week lol.